<launch>

  <!-- A simple launch file for the urg_node package. -->

  <!--  When using an IP-connected LIDAR, populate the "ip_address" parameter with the address of the LIDAR.
      Otherwise, leave it blank. If supported by your LIDAR, you may enable the publish_intensity
      and/or publish_multiecho options. -->

  <arg name="param_file" default="$(find-pkg-share urg_node)/launch/urg_node_serial.yaml" />

  <node name="urg_node" pkg="urg_node" exec="urg_node_driver" output="screen">
    <param from="$(var param_file)" />
  </node>

</launch>
